### DISCUSSION TITLE
9.6.2023

### DISCUSSION THREAD

**Discussion by charlieforward9** [charlieforward9/ATHLEAT#2] [Team Meetings]
 data,SDK and apps are very challenging to work with on Windows. iOS is out of the picture.

We will use platform APIs to track data we can all track with our devices.

### Tracking:
   * **Fitness**: will be tracked with manual recordings on Strava, accessed through their API
   * **Nutrition**: will be tracked with manual inputs to MyFitnessPal, accessed through their API
   * **Mood**: will be tracked through our platform and manually input on a recurring basis.

### Frontend: Typescript with React
* Calls API from the backend with data prepared to be visualized

### Backend: Python
* Call API from services and process data into simple shapes for performant storage and visualization.

### Infrastructure/Tooling: AWS Ecosystem 
* _Breakdown of Database/Hosting/API Services TBD_ 

Meeting with the professor on Tuesday morning 9:30 AM

## Takeaway Tasks
* Research Architecture/Tooling best suited for our **data aggregation** & **visualization** use-case

**Comment by charlieforward9** [comment]
API Links
* [Strava](https://developers.strava.com/)
* [MyFitnessPal](https://myfitnesspalapi.com/)

**Comment by charlieforward9** [comment]
Attendees @rensaketh @PrestonGoren @ticogarcia @charlieforward9

**Reply by charlieforward9** [reply]
The professor just responded, he's got a hard deadline next Tuesday at 10AM. Let's plan on meeting him at 9AM. We will meet through Zoom, link sent ahead of meeting.

**Reply by rensaketh** [reply]
Sounds good!

**Comment by charlieforward9** [comment]
Interesting API Framework: https://github.com/tiangolo/fastapi
